File: Challenges/Coding World Cup/2023/Day 20 - SA vs BAN - Best Team.java
import java.util.Arrays;

public class Solution {
    static long gunDevil(int n, int[] a) {
        Arrays.sort(a);
        long ans = a[n - 1];
        for (int i = 0; i < n - 2; i++) {
            ans += a[i];
        }
        return ans;
    }
}

File: Challenges/Coding World Cup/2023/Day 20 - SA vs BAN - Best Team.cpp
long long gunDevil(int n, vector<int> &a)
{
    // Write your code here.
    sort(a.begin(), a.end());

    long long ans = a[n - 1];

    for (int i = 0; i < n - 2; i++)
        ans += a[i];

    return ans;
}

